### ISSUE TITLE
Right-pointing red triangles doesn't display anything

### ISSUE THREAD

**Issue by ithinkicancode** [weirane/vscode-errorviz#2]
Have "rust-analyzer.diagnostics.useRustcErrorCode" set to true but right-clicking (or any related commands) doesn't do anything. Doesn't show any viz like what is shown in the PDF. Also, in the readme of this repo, some pictures of how the viz looks like will be great as I have no idea what to expect. Even the picture in the PDF is vague, like is that what actually shows in vscode or is that just some "modified/edited" version of the visualization?

BTW, I used the same example in the PDF to get the lifetime error.

**Comment by weirane** [comment]
Maybe the term in the readme is confusing. By "Right-pointing red triangles" I really mean "triangles that point to the right", not clicking the triangles. You need to first move the text cursor to the line with the triangle and use ctrl+shift+v to get the visualization. vscode doesn't support clickable gutters at the moment so we couldn't add clicking functionality on the triangles.

Adding pictures in readme sounds good; I'll try to add them. The figures in the PDF are actual screenshots of vscode.

**Comment by ithinkicancode** [comment]
Yeah, I tried placing the cursor on the offending line and triggering the Toggle Viz command but I don't see any viz :) I used the same code example in the PDF and I did see the red triangle. The Toggle command seems to just change the direction of the red triangle - it appears it supposedly expands/collapses a div (I'm guessing here).

**Comment by Ten0** [comment]
That might be a different issue, but when triggering revis on a lifetime error, all I get is:

![image](https://github.com/weirane/vscode-revis/assets/9094255/3e49edb8-7a98-4bb1-b435-fd36fcdd553c)

**Comment by weirane** [comment]
> That might be a different issue, but when triggering revis on a lifetime error, all I get is:
> 
> ![image](https://user-images.githubusercontent.com/9094255/270417068-3e49edb8-7a98-4bb1-b435-fd36fcdd553c.png)

What Rust code are you testing with and what's your compiler version? There may be an error format that we missed.

**Comment by Ten0** [comment]
I don't recall what code it was. It's always latest compiler version.
It may have been a complex lifetime error.
It would probably be useful if the error gave information about what it got and what it expected when it says it can't parse. That would likely make it easier to debug.
